### ISSUE TITLE
Update our command line tooling for external use

### ISSUE THREAD

**Issue by Yawning** [oasislabs/oasis-core#1929]
Before third parties will deploy nodes, the ekiden command line tooling needs major improvements.

 * [x] There needs to be a sub-command that allows the initialization of a node (so that a node identity is generated), so it can be added to the entity descriptor (#399).  The genesis validator provisioning sub-command should be folded into this.
 * [x] There could be a sub-command that allows editing the entity descriptor (though I'm partially of the opinion that `/bin/vi` is sufficient) so that the node identities can be added to the node list (#399).
 * [x] There needs to be tooling for generating a key manager policy, and signing it.
 * [x] There needs to be a tool for interacting with the staking banking code (#2002).
 * [ ] Other things I'm probably forgetting here.

This probably has overlap with other epics (Staking, KM backfill, etc), but collecting the tooling related issues in one place seems prudent.  Note that testing most of this stuff is  basically blocked on the new and improved deployment process, and/or #1810.

**Comment by kostko** [comment]
Anything that we still need to do and doesn't have an issue filed?

**Comment by Yawning** [comment]
If there is more things to be done, we can file new issues, we have a KM policy tool so, I'll call this done for now.
